rel-eng/tito.props

```diff
@@ -460,62 +467,238 @@ whitelist = gyp
   rubygem-activemodel
   rubygem-activepack
   rubygem-activerecord
+  rubygem-activerecord-session_store
   rubygem-activestorage
   rubygem-activesupport
   rubygem-activeview
+  rubygem-addressable
+  rubygem-algebrick
+  rubygem-ancestry
   rubygem-ansi
+  rubygem-ansi
+  rubygem-apipie-bindings
+  rubygem-apipie-params
+  rubygem-apipie-rails
   rubygem-arel
+  rubygem-audited
+  rubygem-autoprefixer-rails
+  rubygem-awesome_print
+  rubygem-bcrypt
+  rubygem-bootstrap-sass
   rubygem-builder
+  rubygem-bundler_ext
+  rubygem-clamp
+  rubygem-clamp
   rubygem-coffee-rails
   rubygem-coffee-script
   rubygem-coffee-script-source
   rubygem-concurrent-ruby
-  rubygem-clamp
+  rubygem-concurrent-ruby-edge
+  rubygem-connection_pool
   rubygem-crass
+  rubygem-css_parser
+  rubygem-css_parser
```

Comment:
Lots of duplicated entries here. The linked script above sorts and filters out duplicates.